airbyte-integrations/connectors/source-stripe/source_stripe/schemas/subscriptions.json

```diff
@@ -11,7 +11,7 @@
     "livemode": {
       "type": ["null", "boolean"]
     },
-    "start": {
+    "start_date": {
```

Comment:
I'm interested to know why a failing test didn't catch this before?  Is it because we have yet to test this stream with expected_records?

Reply:
We can potentially test it with expected records if we strictly compare records. If not, then we allowed by default to have additional fields that not specified in schema. We currently have task to migrate to stripe tests to strict level (assigned to me) so I can check it during migration

Reply:
> We currently have task to migrate to stripe tests to strict level (assigned to me) so I can check it during migration

Does this imply that we _intend_ to have all the fields in the schema, but we just missed this one? (And the future test work will help us catch it faster?)

Reply:
I just have remembered that migration did not catch it properly with expected records :)

Reply:
> I just have remembered that migration did not catch it properly with expected records :)

Can you please file a bug about this then?

Reply:
@evantahler @mfsiega-airbyte I'm not sure this is actually a bug. A missing field may be not spotted by the test because we guarantee that `additionalProperties` field always equals `True`. Do you think we should override this requirement when testing if data matches its schema? if yes, I will create an issue for that.
Another thing is that we have lots of streams not filled with data so the test is just skipped. This should be fixed for GA connectors when they all have a test strictness level set to `high`